src/libraries/Pool.sol

```diff
@@ -237,9 +237,9 @@ library Pool {
         }
     }
 
-    // the results after a swap, written to storage and returned
+    // Tracks the state of a pool throughout a swap, and bubbles these values up at the end of the swap
     struct SwapResult {
-        // the final current sqrt(price)
+        // the final sqrt(price)
```

Comment:
i'd rather it was just `current`? Its not just final, its current throughout the swap